microhapdb/__init__.py

```diff
@@ -43,10 +43,10 @@ def retrieve_by_id(ident):
 
     >>> retrieve_by_id("mh17KK-014")
                 Name  NumVars  Extent  Chrom    Start      End                Positions              Positions37                          RSIDs    Source     Ae
-    1979  mh17KK-014        3      37  chr17  4497061  4497097  4497061;4497089;4497097  4400356;4400384;4400392  rs333113;rs8074965;rs11657785  Kidd2018  3.923
+    2378  mh17KK-014        3      37  chr17  4497061  4497097  4497061;4497089;4497097  4400356;4400384;4400392  rs333113;rs8074965;rs11657785  Kidd2018  3.923
     >>> retrieve_by_id("rs8074965")
                 Name  NumVars  Extent  Chrom    Start      End                Positions              Positions37                          RSIDs    Source     Ae
-    1979  mh17KK-014        3      37  chr17  4497061  4497097  4497061;4497089;4497097  4400356;4400384;4400392  rs333113;rs8074965;rs11657785  Kidd2018  3.923
+    2378  mh17KK-014        3      37  chr17  4497061  4497097  4497061;4497089;4497097  4400356;4400384;4400392  rs333113;rs8074965;rs11657785  Kidd2018  3.923
```

Comment:
In addition to updating tallies, many of the changes to the test suite involved minor updates to e.g. these indices.